sklearn/utils/multiclass.py

```diff
@@ -62,7 +62,7 @@ def unique_labels(*ys):
 
     Parameters
     ----------
-    ys : array-likes,
+    *ys : array-likes,
```

Comment:
Yea it does... This function is not documented, however similar changes in `shuffle` correctly render the html as shown below...

![image](https://cloud.githubusercontent.com/assets/9487348/5654241/2fc0cc2a-96e7-11e4-9417-eee0f4efac1b.png)